service/src/test/java/bio/terra/externalcreds/pact/VerifyServicePacts.java

```diff
@@ -48,9 +51,24 @@
     properties = {"server.port=8080"},
     classes = ExternalCredsWebApplication.class)
 @ActiveProfiles({"test", "human-readable-logging"})
-@au.com.dius.pact.provider.junitsupport.Provider("ecm")
+@au.com.dius.pact.provider.junitsupport.Provider("externalcreds")
 @PactBroker
 public class VerifyServicePacts {
+  private static final String CONSUMER_BRANCH = System.getenv("CONSUMER_BRANCH");
+  @PactBrokerConsumerVersionSelectors
+  public static SelectorBuilder consumerVersionSelectors() {
+    // The following match condition basically says
+    // If verification is triggered by Pact Broker webhook due to consumer pact change, verify only
+    // the changed pact.
+    // Otherwise, this is a PR, verify all consumer pacts in Pact Broker marked with a deployment
+    // tag (e.g. dev, alpha).
+    if (StringUtils.isBlank(CONSUMER_BRANCH)) {
+      return new SelectorBuilder().mainBranch().deployedOrReleased();
```

Comment:
Try use following (see [https://docs.pact.io/implementation_guides/jvm/provider/junit#selecting-the-pacts-to-verify-with-consumer-version-selectors-4314](https://docs.pact.io/implementation_guides/jvm/provider/junit#selecting-the-pacts-to-verify-with-consumer-version-selectors-4314)). The reason being: the `cbas-externalcreds` pact was published with the cbas PR branch so far and there is neither a published pact associated with the `cbas` main branch nor an associated environment (dev, staging, or production).
```suggestion
      return new SelectorBuilder().mainBranch().deployedOrReleased().branch("WM-2554_ecm-pact-test-tweaks", "cbas");
```

Reply:
thanks @ichengchang ! this worked when I ran the verification code locally, with hardcoded broker credentials. I think I'm going to get the CBAS side merged in and then re-run the code that's already here, rather than put in this workaround only to take it out later